private/bufpkg/bufconfig/generate_plugin_config.go

```diff
@@ -282,9 +275,6 @@ func newPluginConfigFromExternalV1Beta1(
 func newPluginConfigFromExternalV1(
 	externalConfig externalGeneratePluginConfigV1,
 ) (GeneratePluginConfig, error) {
-	if externalConfig.Remote != "" {
```

Comment:
Can you also search all "alpha plugin" references and clean them up (likely comments), such as line 310 in this file